### ISSUE TITLE
Missing accessiblity attributes in UI components

### ISSUE THREAD

**Issue by jreznot** [cuba-platform/cuba#947]
### Environment

- Platform version: 6.9.0

- Client type: Web

### Description of the bug or enhancement

Missing accessibility attributes:

1. Page - title attribute
2. Menubar (note, these changes only worked for the top level items, because their children don’t exist at load)
    role of menuItem on all items
    aria-label on all menu items
3. Filter
    aria-labelledby attribute to link maxResultsLabel to maxResults fields (textfield and button)
    aria-label on filtersPopupButton
    aria-label on settingsBtn
4. Table headers
5. DataGrid
    summary attribute on table
    scope attribute on all header cells
    aria-label on context menu
6. Tab Panel
    aria-labelledby to current tab’s label ID
7. Date Fields
    aria-labelledby to link text input to field’s label
